src/esn.calendar.libs/app/components/time-select/time-select-scroll.directive.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('esn.calendar.libs')
+  .directive('timeSelectScroll', timeSelectScroll);
+
+function timeSelectScroll($timeout) {
+  return {
+    restrict: 'A',
+    link: function($scope, element) {
+
+      $scope.$on('$mdMenuOpen', function() {
+        const target = element[0].querySelector('.active-item');
+
+        $timeout(function() {
+          target.scrollIntoView();
+        }, 500);
```

Comment:
Since using timeout like this is not always recommended, we need a comment to explain why this is needed.